src/messages.cs

```diff
@@ -0,0 +1,286 @@
+//
+// Messages bindings
+//
+// Authors:
+//	Vincent Dondain <vincent@xamarin.com>
+//
+// Copyright 2016 Xamarin Inc. All rights reserved.
+//
+
+using System;
+using XamCore.CoreFoundation;
+using XamCore.CoreGraphics;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+using XamCore.UIKit;
+
+#if !MONOMAC
+namespace XamCore.Messages {
+
+	[iOS (10,0)]
+	[Native]
+	public enum MSMessagesAppPresentationStyle : nuint
+	{
+		Compact,
+		Expanded
+	}
+
+	[iOS (10,0)]
+	[Native]
+	public enum MSStickerSize : nint
+	{
+		Small,
+		Regular,
+		Large
+	}
+
+	[iOS (10,0)]
+	[BaseType (typeof(UIViewController))]
+	interface MSMessagesAppViewController
+	{
+		// inlined ctor
+		[Export ("initWithNibName:bundle:")]
+		[PostGet ("NibBundle")]
```

Comment:
That `[PostGet]` is not needed anymore since we have deprecated/removed non-NRC builds.